src/common/Annotations.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

Reply:
Technically this file was copied from another project (corefx, now dotnet/runtime) and this was the original copyright header. They can be thought of as part of the framework rather than the library, and so it makes sense that they share the same license as the framework IMO.

Reply:
> Technically this file was copied from another project (corefx, now dotnet/runtime) and this was the original copyright header.

Impossible. corefx always uses the MIT banner. Even Roslyn which is licensed under Apache uses the MIT banner for this file.

Reply:
My mistake. It was copied from dotnet/project-system which is Apache 2.0. I'll update the license there too.